docker-compose.yml

```diff
@@ -0,0 +1,204 @@
+# This is a Docker Compose development environment for Restarters.  It:
+# - relies on you setting up www.example.com in your hosts file.  TODO this name could be better but needs some configuration.
+# - a web server on http://www.example.com:8001 which serves up restarters.  Admin user is jane@bloggs.net/passw0rd.
+# - phpMyAdmin on http://www.example.com:8002.  Use host restarters_db, user root/s3cr3t
+# - Discourse on http://www.example.com:8003.  User is someuser/mustbetencharacters; see below about hostname.
+#   TODO This is stock Discourse - no theme.  Would be nice to have the theme, but that doesn't matter so far as
+#   testing the integration.
+# - Mailhog on http://localhost:8025.
+#
+# TODO If your host is Linux, this probably only works if you run as root.
+#
+# First set www.example.com in your hosts file to whatever your host is - e.g. localhost or a VM.
+#
+# Then start using:
+# - docker-compose up -d
+#
+# The Restarters web server won't be available for a couple of minutes after that finishes - it will be doing a composer
+# install and artisan migrate. Discourse also takes quite a few minutes to finish initialising.
+#
+# Once Discourse is up then log in using someuser/mustbetencharacters, and:
```

Comment:
Suggest change to:

> Once Discourse is up, then:

 - Log in to http://restarters.docker:8003 using using someuser/mustbetencharacters
 - etc etc same as currently there

Reply:
Will do example change of domain.